### DISCUSSION TITLE
Runtime Error Dream Texture Animation

### DISCUSSION THREAD

**Discussion by jeffyspoke** [carson-katri/dream-textures#633] [Q&A] [answered]
I've been using different prompts on Dream Texture 0.1.0 in Blender 3.4.1 to get something to pop out of the default cube, but it fails on the first frame:

`RuntimeError: Given groups=1, weight of size [320, 5, 3, 3], expected input[2, 4, 66, 66] to have 5 channels, but got 4 channels instead`

Anybody know how to resolve this? Thank you

**Comment by NullSenseStudio** [comment, marked as answer]
Looks like you're using some conflicting settings. You've only got color pass input while having a depth model selected. You'll either need to use a standard model or change pass inputs to depth or depth and color.

**Reply by jeffyspoke** [reply]
I used Depth and it worked - thanks

**Comment by jeffyspoke** [comment]
Ok - thanks - I'll check it out
